db/migrate/20230920210641_adding_foreign_keys_to_follows.rb

```diff
@@ -0,0 +1,10 @@
+class AddingForeignKeysToFollows < ActiveRecord::Migration[7.0]
+  def change
+    #Adding new columns for following and follower
+    add_column :follows, :following_user_id, :integer
+    add_column :follows, :follower_user_id, :integer
+    #Adding foreign keys to table follows making a reference with table users
+    add_foreign_key :follows, :users, column: :following_user_id
+    add_foreign_key :follows, :users, column: :follower_user_id
```

Comment:
All of this fields could have been added inside this migration why they were not included there?

Reply:
To be honest, I realized I had to add those foreign keys just after I ran the db:migrate in the first migration. I know it doesn’t make sense to add a new empty migration and then creating another when I could simply add those in the first one, but it was a mistake I made😿